api/avatar.go

```diff
@@ -83,7 +83,7 @@ func HandleAddAvatar(ctx *fasthttp.RequestCtx) (interface{}, error) {
 	}
 
 	if i > 0 {
-		toLogCandidate(ctx, DB, id, "avatar", CODE_LOG_UPDATE)
+		toLogCandidate(ctx, DB, id, "avatar", db.GetLogUpdateId())
```

Comment:
I think that we need several functions which must doing different log operations.
toLogCandidateUpdate
toLogCandidateAdd
& others
 Th function must use constant within its body